src/stories/Counter.stories.js

```diff
@@ -0,0 +1,14 @@
+import { Counter } from "../shared/ui/";
+
+export default {
+  title: "Counter",
+  component: Counter,
+};
+
+const Template = (args) => <Counter {...args} />;
+
+export const Default = Template.bind({});
+
+Default.args = {
+  initialValue: 10,
```

Comment:
if we have another view for empty state, we can also add it to storybook

Reply:
done